internal/tokenizer/tokenize_c.go

```diff
@@ -0,0 +1,16 @@
+// +build flex
+
+package tokenizer
+
+import "gopkg.in/src-d/enry.v1/internal/tokenizer/flex"
+
+// Tokenize returns language-agnostic lexical tokens from content. The tokens
+// returned should match what the Linguist library returns. At most the first
+// 100KB of content are tokenized.
```

Comment:
In order to avoid skew, maybe we should export the constant:

```go
// ByteLimit defines the maximum prefix of an input text that will be tokenized for classification.
const ByteLimit = 100000
…
// Tokenize returns lexical tokens from content. The tokens returned match what
// the Linguist library returns. At most the first ByteLimit bytes of content are tokenized.
```

YMMV.

Also possibly worthy of note, splitting at a byte offset means we might partition a multibyte unicode character in the middle of a token, but I think that probably doesn't matter in most cases—the only concern is if that results in a false shibboleth for some language, but I think that is probably just theoretical.